sample-app/redisearch-node-rest/package.json

```diff
@@ -3,32 +3,35 @@
   "version": "1.0.0",
   "description": "Sample RediSearch REST Server with Node.js",
   "main": "server.js",
+  "type": "module",
   "scripts": {
-    "start": "node server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "./node_modules/nodemon/bin/nodemon.js",
```

Comment:
npm adds "./node_modules/.bin" to the path, "nodemon" should work (without the all path)